modules/ROOT/pages/index.adoc

```diff
@@ -1,28 +1,60 @@
 
-[NOTE]
-xref:cloudhub.adoc[CloudHub] is a complete integration platform as a service (iPaaS) that covers all of your server needs out of the box, including Object Store infrastructure, Monitoring, Troubleshooting, Scheduling, and so on. Use Runtime Manager to deploy to CloudHub.
+Anypoint Runtime Manager is the interface to https://anypoint.mulesoft.com[Anypoint Platform]
+that provides a unified view of your applications, servers, and APIs.
+
+With Runtime Manager, you can deploy, manage, and monitor your Mule applications in a sandbox, staging,
+or production environment from one central location, whether your apps are deployed in the cloud or on-premises.
+
+You can use Runtime Manager in these scenarios: 
+
+CloudHub::
+MuleSoft hosts your app deployments in CloudHub and you use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
++
+CloudHub is a complete integration platform as a service (iPaaS) that covers all of your server needs out of the box, including object store infrastructure, monitoring, troubleshooting, and scheduling.
 
-== Architecture
+Hybrid::
+You host your app deployments on your on-premises Mule servers and use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them.
 
-The cloud-based version of the Anypoint Platform has a Runtime Manager with a console that allows you to manage applications deployed to xref:cloudhub.adoc[CloudHub] or elsewhere. 
+Anypoint Platform Private Cloud Edition::
+You host your app deployments on your local Mule servers and use the on-premises Runtime Manager instance 
+included with Anypoint Platform PCE to manage them.
++
+If you are using the on-premises Runtime Manager, you can deploy anywhere _except_ CloudHub.
 
-If you are using the on-premises Runtime Manager you can deploy anywhere _except_ CloudHub.
+Anypoint Runtime Fabric::
+You host your app deployments on your locally installed Runtime Fabric instance and use the https://anypoint.mulesoft.com[Anypoint Platform] Runtime Manager cloud console to manage them. 
 
-For applications deployed to CloudHub (Cloud runtime in the diagram below) you can use xref:insight.adoc[Insight] for analytics and monitoring. For applications that you deploy on-premises or to your private cloud servers you can xref:sending-data-from-arm-to-external-analytics-software.adoc[send data to third-party Analytics applications].
+// image::arm-big-picture.png[]
+image::rtm-overview.png[]
 
-image::arm-big-picture.png[]
+For applications deployed to CloudHub, you can use Insight for analytics and monitoring.
 
-Runtime Manager provides a dashboard that indicates the status of every deployed app and monitored server, with a rich set of features you can use to manage and track activity of your running applications.
+For applications that you deploy on-premises or to your private cloud servers, you can send data to third-party analytics applications.
```

Comment:
Would there be any advantage to your audience to have the analytics info in each of the above descriptions?